test/test.ts

```diff
@@ -1,5 +1,26 @@
-describe('this package', () => {
-  it('should have tests', () => {
-    // Write your test
-  });
+import { getRelHref } from '../src/user/service';
+import { expect } from 'chai';
+
+describe('URI Test', () => {
+
+  const expectedValue = '/user/1'
+
+    it('should pass with full url passed in', () => {
+      const url = 'http://test.com/user/1';
+      const result = getRelHref(url);
+      expect(result).to.equal(expectedValue);
+    });
+
+    it('should pass with relative url being passed in', () => {
+      const url = '/user/1';
+      const result = getRelHref(url);
+      expect(result).to.equal(expectedValue);
+    });
+
+    it('should fail if URL doesn`t contain the opening /', () => {
+      const url = 'user/1';
+      const result = getRelHref(url);
+      expect(result).to.not.equal(expectedValue);
```

Comment:
Usually if a test is described as something like 'should fail', I think my assumption is that an exception is thrown.

However, it here it seems that should fail' is interpreted as 'return the wrong result'. That doesn't sound right to me